app/services/food_service.rb

```diff
@@ -0,0 +1,8 @@
+class FoodService
+  def self.foods_by_ingredient(ingredient)
+    search = "sweet potatoes"
```

Comment:
I know I mentioned this before, but if in your controller you grabbed `params[:q]`, then you would have access to the search term that was used, and you wouldn't need to hard code this.